src/status_im/ui/screens/add_new/models.cljs

```diff
@@ -4,16 +4,24 @@
             [status-im.i18n :as i18n]
             [status-im.ui.screens.navigation :as navigation]
             [status-im.utils.universal-links.core :as universal-links]
-            [status-im.utils.fx :as fx]))
+            [status-im.utils.fx :as fx]
+            [status-im.browser.core :as browser]
+            [status-im.constants :as constants]
+            [status-im.extensions.core :as extensions]))
+
+(defn- valid-url? [url]
```

Comment:
What kind of URL do we validate here?

Reply:
it uses `regx-url` defined in `status-im.constants`

Reply:
Probably cleaner to have this logic in a dedicated namespace. Do we have a relevant one already?

Reply:
let me know where this helper function should be placed and I will move it. Otherwise i could just have it's implementation within the `cond` statement itself and not define any helper function.